wp-includes/class-wp-customize-manager.php

```diff
@@ -25,7 +25,6 @@ function pwa_customize_register_site_icon_maskable( WP_Customize_Manager $wp_cus
 		array(
 			'capability' => 'manage_options',
 			'type'       => 'option',
-			'default'    => false,
```

Comment:
Was this causing a problem?

Reply:
It was causing static analysis error as default arg is string.

Reply:
See [job](https://github.com/GoogleChromeLabs/pwa-wp/actions/runs/4107427913/jobs/7345812124) for runner logs.

https://github.com/WordPress/WordPress/blob/6d0eaf9674e550788dce7942d87af0e4f0147784/wp-includes/class-wp-customize-setting.php#L62-L68